video_xblock/static/html/brightcove.html

```diff
@@ -1,9 +1,16 @@
 <style>
+    div.vjs-control-bar > div.vjs-menu-button.vjs-menu-button-popup.vjs-control.vjs-button.icon.fa.fa-cc.vjs-singleton > div > ul,
```

Comment:
i think it is a better to remove all styles from HTML that does not include
```{{player_id}}```

Reply:
@otecMuxah so, please, remove all styles you think should be removed.